src/comps/apy/calc.js

```diff
@@ -29,11 +29,14 @@ function calcApyPerAsset({ rewardsInfo, nftinfo }) {
         if (nftRewardsAmt === 0) {
             nftinfo[i].apy = 0
             nftinfo[i].apr = 0
+            nftinfo[i].roundYield = 0
             return
         }
-        const nftApr = nftRewardsAmt / (nft.ocean_allocated + nft.ocean_allocated_owner)
-        const nftApy = wprToApy(nftApr)
-        nftinfo[i].apr = nftApr
+        const nftRoundYield =
+            nftRewardsAmt / (nft.ocean_allocated + nft.ocean_allocated_owner)
+        const nftApy = wprToApy(nftRoundYield)
+        nftinfo[i].roundYield = nftRoundYield
+        nftinfo[i].apr = nftRoundYield * 52
```

Comment:
I see we're now calculating and reporting both properly.

Reply:
I also see that you just added roundYield rather than remove anything, so I'm going to reconcile this with df-web